frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -279,6 +279,8 @@ export const surveyLogic = kea<surveyLogicType>([
                     `,
                 }
                 const responseJSON = await api.query(query)
+                // TODO:Dylan - I don't like how we lose our types here
```

Comment:
I need to link this to an issue and chuck it in our backlog; it's not high-priority, but it would be lovely to have principled parsing here – it would've fixed the bug I resolve a few lines later.